composer.json

```diff
@@ -13,12 +13,13 @@
     }
   ],
   "require": {
-    "php": ">=7.0",
+    "php": "~7.1",
```

Comment:
incorrect version range cause your value is  `<7.2` and can broke some installations on 7.2
I prefer leave old value `>=7.0`